packages/firebase_storage/lib/firebase_storage.dart

```diff
@@ -33,6 +34,16 @@ class StorageReference {
     task._start();
     return task;
   }
+
+  Future<Uint8List> getData(int maxSize) {
```

Comment:
I realize that that plugin doesn't have dartdocs and tests yet. If you feel like writing some (at least for the code you're adding) it would be appreciated.